### ISSUE TITLE
internal diameter of tibia

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3806]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. decreased internal diameter of tibia
2. increased internal diameter of tibia

>2. Synonyms (e.g., absent spleen)

1. decreased endosteal diameter of tibia ;   decreased tibia bone marrow cavity diameter ;   decreased tibia medullary cavity diameter ;   decreased inner diameter of tibia ;   reduced endosteal diameter of tibia ;   reduced inner diameter of tibia
2. increased endosteal diameter of tibia ;   increased tibia bone marrow cavity diameter ;   increased tibia medullary cavity diameter ;   increased inner diameter of tibia

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. reduced cross-sectional distance that extends from one lateral edge of the tibia marrow cavity, through its center and to the opposite lateral edge of the bone marrow cavity through the mid point of the tibia
2. increased cross-sectional distance that extends from one lateral edge of the tibia bone marrow cavity, through its center and to the opposite lateral edge of the bone marrow cavity through the mid point of the tibia

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:35239186

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal long bone internal diameter (MP:0013619)
2. abnormal long bone internal diameter (MP:0013619)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5356-4174

**Comment by sbello** [comment]
Added
increased - MP:0014292
decreased - MP:0014293
